lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from "prop-types"
 
 const Cards = props => {
   return (
     <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+    {props.cards.map((card,index) => (
+      <Card
+        key={index}
+        card={card}
+        headline={card.headline}
```

Comment:
Note that you are passing all the card info on the card prop and don't need to break out the individual values here, you can do that in the Card component